src/plugins/tracing.ts

```diff
@@ -80,26 +61,34 @@ export function filterFilmstrips(buffer: Buffer): Array<FilmStrip> {
  */
 export class Tracing {
   async start(client: CDPSession) {
-    const includedCategories = ['disabled-by-default-devtools.screenshot'];
-    const { categories } = await client.send('Tracing.getCategories');
-    const excludedCategories = categories.filter(
-      cat => !includedCategories.includes(cat)
-    );
-
+    const includedCategories = [
+      // exclude all default categories
+      '-*',
+      // capture screenshots
+      'disabled-by-default-devtools.screenshot',
+    ];
     await client.send('Tracing.start', {
-      transferMode: 'ReturnAsStream',
+      transferMode: 'ReportEvents',
```

Comment:
The transferMode `ReportEvents` is the default mode, so we can remove this.
also, might be worth adding a comment in the code to explain choosing the default mode over `ReturnAsStream`

Reply:
Yeah its the default, i wanted to keep it as they are deprecating some of the fields and might change the default anytime. I will add a comment though about the perf implication.

Reply:
Out of interest, do you have a link for the that announcement?

Reply:
No i meant the deprecation notices in the latest tip of tree in CDP https://chromedevtools.github.io/devtools-protocol/tot/Tracing/#method-start. None of them are at the stable RC at the moment. Anything could change, so this was just us being defensive, nothing announced yet.